c/models/column_hasher.cc

```diff
@@ -20,17 +20,18 @@
 // IN THE SOFTWARE.
 //------------------------------------------------------------------------------
 #include "models/column_hasher.h"
+#include <float.h>
 
 
 /*
-* Abstract Hasher class constructor and destructor.
+*  Abstract Hasher class constructor and destructor.
```

Comment:
FYI, I usually prefer to indent these comments as
```
/**
 * content...
 */
```
This way if the comment gets big enough it can be collapsed in Sublime Text.
You may, of course, use your own style if you prefer it that way.

Reply:
Sure, let's convert all the comments to the format Sublime understands. Btw, any reason you sometimes use different format? Like [here](https://github.com/h2oai/datatable/blob/master/c/writebuf.cc#L32-L63), for example.

Reply:
No reason, just happens >.>

Reply:
np :) I converted it to the above format just adding one more space for indentation, as it looks to me that then `*` is better separated from the content. Still ok for Sublime.